content/en/docs/setup/install/operator/index.md

```diff
@@ -0,0 +1,122 @@
+
+- Descriptive error messages to spot configuration errors easier.
+
+- Installer capable of waiting for Istio pods and services to be in a
+  ready state.
+
+- Fewer dependencies (doesn’t require Helm).
+
+The Operator install is accessed via [`istioctl`](/docs/reference/commands/istioctl/)
+commands and a new Operator API.
+
+## Prerequisites
+
+Before you begin, you must meet the following prerequisites.
+
+1. [Download the Istio release](/docs/setup/#downloading-the-release).
+1. Perform any necessary [platform-specific setup](/docs/setup/install/platform/).
+1. Check the [Requirements for Pods and
+   Services](/docs/setup/additional-setup/requirements/).
+
+## Install Istio using One Line
+
+The simplest option is to install Istio using a one-line command. At the command
+prompt, run the following command:
+
+{{< text bash >}}
+$ istioctl manifest apply
+{{< /text >}}
+
+This command installs a profile named `default` on the cluster defined by your
+Kubernetes configuration. The `default` profile is smaller and more suitable
+for establishing a production environment, unlike the larger profile named
+`demo` that is meant to evaluate a broad set of Istio features.
+
+You can override the `default` profile using the `--kubeconfig` flag.
+
+You can view the profiles named `default`, `demo`, and others in the
+[operator GitHub repository](https://github.com/istio/operator/tree/master/data/profiles).
+Alternatively, you can display the contents of the `default` profile to the
+console by using the following command:
+
+{{< text bash >}}
+$ istioctl profile dump
+{{< /text >}}
+
+## Install a Different Profile
+
+Other Istio configuration profiles can be installed in a cluster using the
+following command:
+
+{{< text bash >}}
+$ istioctl manifest apply --set profile=<profile name>
+{{< /text >}}
+
+where `<profile name>` is one of the profile names from the output of the
+`istioctl profile list` command.
+
+## Display the Profiles List
+
+You can display the names of Istio configuration profiles that are
+accessible to `istioctl` by using the following command:
+
+{{< text bash >}}
+$ istioctl profile list
+{{< /text >}}
+
+Optionally, you can use the `-s` flag with a install package path to see the
+list of configuration profiles available for other Istio versions. At the
+command prompt, run the following command:
+
+{{< text bash >}}
+$ istioctl profile list -s installPackagePath=https://github.com/istio/istio/releases/tags/1.3.3
+{{< /text >}}
+
+## Inspect/Modify a Manifest Before Installation
+
+You can inspect or modify the manifest before installing Istio using the
+following steps:
+
+1. At the command prompt, run the following command:
+
+{{< text bash >}}
+$ istioctl manifest generate > <file name>
+{{< /text >}}
+
+1. Inspect or modify the manifest as needed.
+1. Then, run the following command:
+
+{{< text bash >}}
+$ kubectl apply -f <file name>
+{{< /text >}}
+
+where `<file name>` is the name of the YAML manifest file.
```

Comment:
delete line or preceed the text with the real life example.

Reply:
addressed